package.json

```diff
@@ -54,5 +54,6 @@
     "access": "public"
   },
   "repository": "https://github.com/open-turo/eslint-config-typescript",
-  "version": "16.0.7"
+  "version": "16.0.7",
+  "packageManager": "npm@11.4.2+sha512.f90c1ec8b207b625d6edb6693aef23dacb39c38e4217fe8c46a973f119cab392ac0de23fe3f07e583188dae9fd9108b3845ad6f525b598742bd060ebad60bff3"
```

Comment:
Sorted `dependencies` + adds `corepack use npm` `packageManager` for better corepack compat.